src/styles/globals.css

```diff
@@ -4,5 +4,17 @@
 @tailwind utilities;
 
 html {
-    font-size: calc(10px + 0.390625vw);
+  font-size: calc(10px + 0.390625vw);
+}
+.ReactModal__Overlay {
+  opacity: 0;
+  transition: opacity 100ms ease-in-out;
+}
+
+.ReactModal__Overlay--after-open {
+  opacity: 1;
+}
+
+.ReactModal__Overlay--before-close {
+  opacity: 0;
 }
```

Comment:
I would prefer having these in the modal props, but this is fine

Reply:
it's so that we can apply these styles globally so that it's consistent across modals without having to write duplicate code. i got this from here https://reactcommunity.org/react-modal/styles/transitions/